components/FeatureHighlight/src/private/MDCFeatureHighlightDismissGestureRecognizer.m

```diff
@@ -86,7 +86,11 @@ - (void)touchesEnded:(NSSet<UITouch *> *)touches withEvent:(UIEvent *)event {
   [super touchesEnded:touches withEvent:event];
 
   if (_hasTouch) {
-    self.state = UIGestureRecognizerStateEnded;
+    if (MDCCGFloatEqual(_progress, 1.0)) {
```

Comment:
I removed that change... I did a bit more research on what the cancelled state means and I think my initial assumption was incorrect.